target/linux/ramips/image/mt7621.mk

```diff
@@ -281,6 +281,16 @@ define Device/cudy_wr2100
 endef
 TARGET_DEVICES += cudy_wr2100
 
+define Device/cudy_x6
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 32448k
```

Comment:
```32256k``` (```0x1F80000```)?

Reply:
I've made all the changes required except this, i don't get what i should do, sorry :(

Before commit and push i want to fix everything

Reply:
Currently, ```IMAGE_SIZE``` is defined as ```32448k``` (= 0x1FB0000) but the size of "firmware" partition is defined as ```0x1f80000``` (= 32256k). ```IMAGE_SIZE``` basically needs to fit the size of "firmware" partition.

Reply:
Got that, thanks for the explanation ur saving my day :D